config/development.yaml

```diff
@@ -21,11 +21,11 @@ services:
   citizen:
     uploadDocsEmail: UPLOAD_DOCUMENTS_EMAIL
   case:
-    url: 'https://ccd-data-store-api-prl-ccd-definitions-pr-942.service.core-compute-preview.internal'
+    url: 'https://ccd-data-store-api-prl-ccd-definitions-pr-1042.service.core-compute-preview.internal'
```

Comment:
@giorgi-hmcts Please avoid these local development changes in the PR.

Reply:
Done. Reverted back to original PR pointers  ✅